### DISCUSSION TITLE
Update Submodule apk-build to support latest v21.4.2 gvmd and other packages

### DISCUSSION THREAD

**Discussion by austinsonger** [Secure-Compliance-Solutions-LLC/GVM-Docker#225] [General]
#219 

<hr /><em>This discussion was created from the release <a href='https://github.com/Secure-Compliance-Solutions-LLC/GVM-Docker/releases/tag/v21.4.2'>Update Submodule apk-build to support latest v21.4.2 gvmd and other packages</a>.</em>

**Comment by Dexus** [comment]
@austinsonger when will this also released to docker?

**Reply by austinsonger** [reply]
We are currently working on our docker build workflow. We are currently resolving some of issues with the pipeline, once we resolve them, we will publish the new build to the Docker container.

**Reply by Dexus** [reply]
Thx would be good to have the Makefile in use to build it, this will be a bit difficult if you do it with the tasks. The image build based on the apk build.
